schemas/Account.json

```diff
@@ -35,6 +35,10 @@
     "is_disabled": {
       "description": "Admin users may disable/enable an account",
       "type": "boolean"
+    },
+    "ledger": {
+      "description": "A link the the account's ledger",
+      "type": "Iri.json"
     }
   },
   "required": ["name", "balance"],
```

Comment:
Good point; unauthorized users will get the account without the balance property.